README.markdown

```diff
@@ -28,7 +28,7 @@ To use a custom timeout, create an initializer file:
 
 ```ruby
 # config/initializers/rack_timeout.rb
-Rack::Timeout.timeout = 5  # seconds
+Rack::Timeout.service_timeout = 5  # seconds
```

Comment:
I'd recommend making this configuration API backward compatible if at all possible (even if it involves introducing a new preferred setting, but still keeping the old one around).